packages/round-manager/src/features/program/ListProgramPage.tsx

```diff
@@ -86,7 +86,7 @@ function ListPrograms() {
         floatingIcon={
           <div
             className="relative flex justify-center items-center h-16 w-16 translate-x-6 -translate-y-8 bg-teal-500 text-white rounded drop-shadow-xl">
-            <RefreshIcon className="h-6 w-6" aria-hidden="true" />
+            <RefreshIcon className="h-6 w-6 text-gray-500" aria-hidden="true" />
```

Comment:
color code should be `text-grey-500` not `text-gray-500`

closely related... ikr